cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -154,6 +154,11 @@
 default['travis_build_environment']['sysctl_disable_ipv6'] = true
 default['travis_build_environment']['wget']['version'] = '1.18'
 
+default['travis_build_environment']['shellcheck_url'] = 'https://s3.amazonaws.com/travis-blue-public/binaries/ubuntu/14.04/x86_64/shellcheck-0.4.4.tar.bz2'
+default['travis_build_environment']['shellcheck_version'] = '0.4.0'
```

Comment:
this is 0.4.0 but the URL is for 0.4.4, and the latest version is 0.4.5. Could we bump them all to 0.4.5?